natsumi/modules/natsumi-base.css

```diff
@@ -114,7 +114,7 @@ their author(s) have been provided above the used code.
 
   &[usercontextid] {
     .tab-background {
-      @media (-moz-bool-pref: "natsumi.sidebar.border-all") {
+      @media (-moz-bool-pref: "natsumi.sidebar.container-tabs-border") {
```

Comment:
@greeeen-dev  - since the config name has changed, you might want to update the `README.md` as well?